CHANGELOG.md

```diff
@@ -9,8 +9,10 @@ To publish a new release run `scripts/release` from the project directory.
 ## [Unreleased]
 
 ### Added
+- Add PowerShell completions (see [GH-184])
```

Comment:
Please add a period at the end and mention the ticket, i.e.

```
- Add PowerShell completions (see [GH-183] and [GH-184]).
```

And please move it _below_ the wezterm change because it has come afterwards.